### PR TITLE
[docs] systemd and config ownership improvements

### PR DESCRIPTION
**Pull request by bmorelli25**
**A preview of the documenation changes in this PR is available [here](http://apm-server_3027.docs-preview.app.elstc.co/diff)**.
Closes https://github.com/elastic/apm-server/issues/2865.

Still some "build" work that needs to be done before this can be merged, mostly in the beats repo.

This PR updates:

* [**Step 3: Start**](https://www.elastic.co/guide/en/apm/server/current/apm-server-starting.html)
Screen Shot 2019-12-10 at 11 12 20 AM

* [**APM Serever and systemd**](https://www.elastic.co/guide/en/apm/server/current/running-with-systemd.html)
Screen Shot 2019-12-10 at 11 16 09 AM

* The command reference documentation, which was linking to the incorrect Beats information.

### CODE REVIEW

File: docs/copied-from-beats/shared-systemd.asciidoc [hunk 1]
```diff
@@ -5,6 +5,11 @@ The DEB and RPM packages include a service unit for Linux systems with
...
 systemd commands.
 
+ifdef::apm-server[]
+We recommend that the {beatname_lc} process is run as a non-root user.
+Therefor, that is the default setup for {beatname_uc}'s DEB package and RPM installation.
+endif::apm-server[]
+
 ==== Start and stop {beatname_uc}
 
 Use `systemctl` to start or stop {beatname_uc}:
```
Comment (bmorelli25) [inline docs/copied-from-beats/shared-systemd.asciidoc]: @graphaelli, any reason why we omit `sudo` from the start/stop/enable/disable commands?

File: docs/config-ownership.asciidoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+-----
+Exiting: error loading config file: config file ("/etc/{beatname_lc}/{beatname_lc}.yml")
+must be owned by the user identifier (uid=1000) or root
+-----
+
+To correct this problem you can change the ownership of the configuration file with:
++chown {beatname_lc}:{beatname_lc} /etc/{beatname_lc}/{beatname_lc}.yml+.
+
+You can also make the config ownership world readable, although this is not recommended:
++chown root:root /etc/{beatname_lc}/{beatname_lc}.yml`+.
```
Comment (axw) [inline docs/config-ownership.asciidoc]: I don't really follow this - the command makes root the owner, it doesn't make it world readable.

Comment (axw) [reply docs/config-ownership.asciidoc]: Oh, the following bits are related to this too? It read to me like the `chown root:root` was meant to make it world readable.

Comment (axw) [inline docs/config-ownership.asciidoc]:
```suggestion
+sudo chown root:root /etc/{beatname_lc}/{beatname_lc}.yml`+.
```

? (though, I note your question to Gil...)

File: docs/copied-from-beats/shared-systemd.asciidoc (hunk 1 above)
Comment (graphaelli) [reply docs/copied-from-beats/shared-systemd.asciidoc]: good question, I'm good with adding `sudo` to make things more copy and paste friendly.

File: docs/config-ownership.asciidoc (hunk 2 above)
Comment (bmorelli25) [reply docs/config-ownership.asciidoc]: Good call. That is confusing.

File: docs/config-ownership.asciidoc [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+-----
+Exiting: error loading config file: config file ("/etc/{beatname_lc}/{beatname_lc}.yml")
+must be owned by the user identifier (uid=1000) or root
+-----
+
+To correct this problem you can change the ownership of the configuration file with:
++chown {beatname_lc}:{beatname_lc} /etc/{beatname_lc}/{beatname_lc}.yml+.
+
+You can also make root the config owner, although this is not recommended:
++sudo chown root:root /etc/{beatname_lc}/{beatname_lc}.yml`+.
```
Comment (axw) [inline docs/config-ownership.asciidoc]:
```suggestion
+sudo chown root:root /etc/{beatname_lc}/{beatname_lc}.yml+.
```

Comment (bmorelli25) [reply docs/config-ownership.asciidoc]: 🙌

**Comment by bmorelli25** [comment]
Thank you! I've merged the required changes in the beats repo in https://github.com/elastic/beats/pull/15126. Will merge this PR when green.

**Pull request merged**